queries.yaml

```diff
@@ -161,21 +161,21 @@ layers:
       - vectordatasource.transform.add_id_to_properties
       - vectordatasource.transform.detect_osm_relation
       - vectordatasource.transform.remove_feature_id
-      - vectordatasource.transform.truncate_min_zoom_to_2dp
+      - vectordatasource.transform.truncate_min_zoom_to_1dp
```

Comment:
All the min_zooms are ints in the area transform below... although switching the NE values to be data driven may sometimes introduce some float values with tenths. So this doesn't hurt, but it probably doesn't help as much now that we're switching to the area lookup table.